app/src/main/java/com/amaze/filemanager/filesystem/HybridFile.java

```diff
@@ -825,12 +747,7 @@ public Void execute(SFTPClient client) {
         try {
           SmbFile smbFile = new SmbFile(path);
           for (SmbFile smbFile1 : smbFile.listFiles()) {
-            HybridFileParcelable baseFile = new HybridFileParcelable(smbFile1.getPath());
-            baseFile.setName(smbFile1.getName());
-            baseFile.setMode(OpenMode.SMB);
-            baseFile.setDirectory(smbFile1.isDirectory());
```

Comment:
Not setting date/size/mode in hybridFile?

Reply:
Should be fine using HybridFileParcelable constructor for SMB files.